sklearn_pandas/features_generator.py

```diff
@@ -0,0 +1,50 @@
+def gen_features(columns, classes=None):
+    """Generates a feature definition list which can be passed
+    into DataFrameMapper
+
+    Params:
+
+    columns     a list of column names to generate features for.
+
+    classes     a list of classes for each feature, a list dictionaries with
+                transformer class and init parameters, or None.
+
+                If list of classes is provided, then each of them is
+                instantiated with default arguments:
+
+                    classes = [StandardScaler, LabelBinarizer]
+
+                If list of dictionaries is provided, then each of them should
+                have a 'class' key with transformer class. All other keys are
+                passed into 'class' value constructor:
```

Comment:
Same here. Say `Example:` to be clear that what we are showing is an example.

Reply:
Got it.